microcosm_pubsub/handlers.py

```diff
@@ -122,3 +126,22 @@ def get_resource(self, message, uri):
 
     def handle(self, message, uri, resource):
         return True
+
+
+@binding("publish_message_batch")
```

Comment:
My thinking is that projects that use the deferred producer can simply require the `publish_message_batch` handler in the appropriate daemon.